gtda/homology/cubical.py

```diff
@@ -141,7 +141,8 @@ def fit(self, X, y=None):
             self.periodic_dimensions_ = np.zeros((len(X) - 1,), dtype=np.bool)
         else:
             self._filtration = PeriodicCubicalComplex
-            self.periodic_dimensions_ = self.periodic_dimensions
+            self.periodic_dimensions_ = np.array(self.periodic_dimensions,
+                                                 dtype=np.bool)
```

Comment:
In light of this [comment](https://github.com/giotto-ai/giotto-tda/pull/333/files#r385162597), i find `np.bool` particularly  worrying.